x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/locked_card.tsx

```diff
@@ -30,7 +30,7 @@ const LockedPolicyDiv = styled.div`
   }
 `;
 
-export const LockedPolicyCard = memo(() => {
+export const LockedPolicyCard = memo(({ title }: { title: React.ReactNode }) => {
```

Comment:
could you add an interface with that type?

Reply:
Do we really need an interface for a type we use once in a single place?

Reply:
Not needed at all, but it gets more readable IMO 🙂

Reply:
I'll skip it for now. if I have to add another parameter I'll change it if you don't mind.